packages/legacy/core/App/navigators/TabStack.tsx

```diff
@@ -46,6 +61,53 @@ const TabStack: React.FC = () => {
     return 0
   }
 
+  const handleDeepLink = useCallback(
+    async (deepLink: string) => {
+      logger.info(`Handling deeplink: ${deepLink}`)
+
+      // If it's just the general link with no params, set link inactive and do nothing
+      if (deepLink.search(/oob=|c_i=|d_m=|url=/) < 0) {
```

Comment:
Is oob or _oob an option also?

Reply:
Yes, `oob=`, `_oob=`, `url=`, `_url=`, `c_i=`, and `d_m=` all get picked up by this check